File: app/src/main/java/com/mieone/feedbackcollection/utils/ActivityManager.java
package com.mieone.feedbackcollection.utils;

import android.content.Context;
import android.content.Intent;

import com.mieone.feedbackcollection.FeedbackActivity;
import com.mieone.feedbackcollection.MainActivity;

import spencerstudios.com.bungeelib.Bungee;

public class ActivityManager {

    public static void HOME(Context context){

        context.startActivity(new Intent(context, MainActivity.class));
    }

    public static void FEEDBACK(Context context, String employee_id, String document_id){

        Intent intent = new Intent(context, FeedbackActivity.class);
        intent.putExtra("employee_id", employee_id);
        intent.putExtra("document_id", document_id);
        context.startActivity(intent);
        Bungee.slideLeft(context);
    }

}

File: app/src/main/java/com/mieone/feedbackcollection/utils/ActivityManager.kt
package com.mieone.feedbackcollection.utils

import android.content.Context
import android.content.Intent

import com.mieone.feedbackcollection.ui.FeedbackActivity
import com.mieone.feedbackcollection.ui.MainActivity

import spencerstudios.com.bungeelib.Bungee

object ActivityManager {

    fun HOME(context: Context) {

        context.startActivity(Intent(context, MainActivity::class.java))
    }

    fun FEEDBACK(context: Context, name: String, manager: String, vendor: String, department: String, doj: String, scanned_id: String, employee_status: String) {

        val intent = Intent(context, FeedbackActivity::class.java)
        intent.putExtra("name", name)
        intent.putExtra("manager", manager)
        intent.putExtra("vendor", vendor)
        intent.putExtra("department", department)
        intent.putExtra("doj", doj)
        intent.putExtra("scanned_id", scanned_id)
        intent.putExtra("employee_status",employee_status)
        context.startActivity(intent)
        Bungee.slideLeft(context)
    }

}
